docs/execute/binaries.md

```diff
@@ -62,6 +121,14 @@ But, obviously, testing validated versions of components is not really interesti
 ## WAF rule set
 
 * copy a file `waf_rule_set` in `binaries/`
+
+#### After Testing with a Custom Tracer:
```

Comment:
Let's make a header "How to Run system-tests" Or something similar and then at the top put this part, but don't make it specific to the Python tracer. We can be somewhat general e.g. _"Note: Most of the ways to run system-tests with a custom tracer version involve modifying the binaries directory. Modifying the binaries will alter the tracer version used across your local computer. Once you're done testing with the custom tracer, ensure you **remove** it. For example for Python: rm -rf binaries/python-load-from-pip"_